<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Createscript</class>
 <widget class="QWidget" name="Createscript">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>458</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Create script</string>
  </property>
  <widget class="QsciScintilla" name="tEScriptEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>601</width>
     <height>301</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pBCreate">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="aaaaLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateDateTimeEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
